wg-apply/README.md

```diff
@@ -11,6 +11,9 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 Improve the state of declarative object management by fixing `kubectl apply`, moving the logic from the client to the server. This move also enables new/fixed features included in the scope of this working-group: diff, dry-run and prune.
 Resources can be found in [this Google drive folder](https://drive.google.com/drive/folders/1wlpgkS2gFZXdp4x2WlRsfUBxkFlt2Gx0)
 
+## Stakeholder SIGs
+* SIG CLI
```

Comment:
Add API Machinery, or omit this section completely (since it's in metadata already).

Reply:
Oh wait, this is a different document. Yes, please add API Machinery.

Reply:
Fixed in ae4388c.

(It's not obvious, but this file is generated from `sigs.yaml`; adding API Machinery to the `stakeholder_sigs` list causes it to be included in this `README.md` automatically.)